apps/melding-form/src/app/(map)/locatie/kies/_components/Map.tsx

```diff
@@ -52,6 +52,38 @@ export const Map = ({ coordinates, showAssetList, setCoordinates }: Props) => {
       ],
     })
 
+    const crosshair = document.getElementById('crosshair')!
+
+    // Set crosshair to show where the marker is set with a keyboard
+    map.on('keydown', ({ originalEvent }) => {
+      if (
+        originalEvent.key === 'ArrowUp' ||
+        originalEvent.key === 'ArrowDown' ||
+        originalEvent.key === 'ArrowLeft' ||
+        originalEvent.key === 'ArrowRight'
+      ) {
+        crosshair.style.display = 'block'
+      }
+      if (originalEvent.key === ' ' || originalEvent.key === 'Enter') {
+        const center = map.getCenter()
+
+        return setCoordinates({
+          lat: center.lat,
+          lng: center.lng,
+        })
+      }
+    })
+
+    // Hide crosshair when map loses focus
+    map.on('blur', () => {
+      crosshair.style.display = 'none'
+    })
+
+    // Hide crosshair when using mouse
+    map.on('mousemove', () => {
```

Comment:
You can combine events, so something like `map.on('blur mousemove', () => {` should work.
We should probably add `dragstart` to that list, to account for touchscreen devices.

Reply:
I was thinking about dragstart but since I am not able to test it I thought I'll leave it out. But added it if you're confident of how it works.